cmd/storj-sim/network.go

```diff
@@ -111,13 +111,14 @@ func newNetwork(flags *Flags) (*Processes, error) {
 
 	processes := NewProcesses()
 	var (
-		configDir       = flags.Directory
-		host            = flags.Host
-		gatewayPort     = 9000
-		bootstrapPort   = 9999
-		satellitePort   = 10000
-		storageNodePort = 11000
-		consolePort     = 10100
+		configDir        = flags.Directory
+		host             = flags.Host
+		gatewayPort      = 9000
+		bootstrapPort    = 9999
+		satellitePort    = 10000
+		storageNodePort  = 11000
+		consolePort      = 10100
+		bootstrapWebPort = 10010
```

Comment:
We need a better strategy for picking these, since they seem really disjointed. But it's fine for now.

Reply:
Maybe let's use `9998` here, less likely to end up conflicting with satellite.